File: app/src/main/java/com/example/movieapi/adapter/MovieItemDiffUtil.java
package com.example.movieapi.adapter;

import androidx.annotation.NonNull;
import androidx.recyclerview.widget.DiffUtil;

import com.example.movieapi.model.MovieItem;

public class MovieItemDiffUtil extends DiffUtil.ItemCallback<MovieItem> {
    @Override
    public boolean areItemsTheSame(@NonNull MovieItem oldItem, @NonNull MovieItem newItem) {
        return false;
    }

    @Override
    public boolean areContentsTheSame(@NonNull MovieItem oldItem, @NonNull MovieItem newItem) {
        return false;
    }
}

File: app/src/main/java/com/example/movieapi/adapter/MovieItemDiffUtil.kt
package com.example.movieapi.adapter

import androidx.recyclerview.widget.DiffUtil
import com.example.movieapi.model.MovieItem

class MovieItemDiffUtil : DiffUtil.ItemCallback<MovieItem>() {

    override fun areItemsTheSame(oldItem: MovieItem, newItem: MovieItem): Boolean =
            oldItem == newItem

    override fun areContentsTheSame(oldItem: MovieItem, newItem: MovieItem): Boolean =
            true
}